genvalidity-hspec-optics/src/Test/Validity/Optics.hs

```diff
@@ -57,7 +57,16 @@ lensSpecOnValid l =
 --
 -- > lensSpec ((_2) :: Lens (Int, Int) (Int, Int) Int Int)
 lensSpec ::
-       forall s b. (Show b, Eq b, GenUnchecked b, Show s, Eq s, GenUnchecked s)
+       forall s b.
+       ( Show b
+       , Eq b
+       , GenUnchecked b
+       , Validity b
+       , Show s
+       , Eq s
+       , GenUnchecked s
+       , Validity s
+       )
```

Comment:
Why are you adding type constraints?

Reply:
Because `lensSpecOnGen` needs more constraints now.